pmd-core/src/main/java/net/sourceforge/pmd/cache/AbstractAnalysisCache.java

```diff
@@ -119,6 +119,8 @@ public void checkValidity(final RuleSets ruleSets, final ClassLoader auxclassPat
 
         final long currentAuxClassPathChecksum;
         if (auxclassPathClassLoader instanceof URLClassLoader) {
+            // we don't want to close our aux classpath loader - we still need it...
+            @SuppressWarnings("PMD.CloseResource")
```

Comment:
That one could be considered a FP, since we don't create the URLClassLoader here....